lib/ext/heroku/command/addons.rb

```diff
@@ -203,7 +203,14 @@ def create
 
       action("Creating #{addon['name'].downcase}") {}
       action("Adding #{addon['name'].downcase} to #{app}") {}
-      action("Setting #{addon['config_vars'].join(', ')} and restarting #{app}") do
+
+      message = ''
+
+      unless addon['config_vars'].empty?
+        message ="Setting #{addon['config_vars'].join(', ')} and restarting #{app}"
+      end
+
+      action(message) do
```

Comment:
@mathias yep, it's used in some display helpers: https://github.com/heroku/heroku/blob/9f8dc3da84835af2b91af22660ed67789950f21d/lib/heroku/helpers.rb#L248-L265